public/app/plugins/datasource/cloudwatch/components/styles.ts

```diff
@@ -50,7 +50,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
     padding: theme.spacing(1, 1, 1, 0),
     width: '25%',
     '&:first-of-type': {
-      width: '50%',
+      width: '80%',
```

Comment:
maybe not that related to the rest of the work in this PR, but wanted to mike first column wider so that there's room to display the full log group name.